pytorch_lightning/callbacks/progress/rich_progress.py

```diff
@@ -156,7 +156,7 @@ def __init__(
         self.theme = theme
 
     @property
-    def refresh_rate(self) -> int:
+    def refresh_rate(self) -> float:
```

Comment:
@SeanNaren Have you experimented with different refresh rates? I found few issues with it.

Reply:
I have not, open an issue to track separately?

Reply:
I realised this comes down to how we treat `refresh_rate` differently for TQDM and for Rich. Will be an issue once we combine the two :)